05-UI-ReactJS/topic-3/src/components/MovieItem.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class MovieItem extends React.Component {
+  render() { //note: always return ONE element
+
+
+    return (
```

Comment:
I'd suggest you to do something like

```
const { title, category, year, duration} = this.props.movie;

...
```

in order to clean up the code and avoid replication of code that is not really needed